lib/api/core/models/repositories/roleRepository.js

```diff
@@ -130,27 +130,20 @@ RoleRepository.prototype.loadRole = function roleRepositoryLoadRole (roleId) {
 /**
  *
  * @param {Array} controllers
- * @param {int} from
- * @param {int} size
  */
-RoleRepository.prototype.searchRole = function roleRepositorySearchRole (controllers, from, size) {
-  var query = {};
-
-  if (controllers && Array.isArray(controllers)) {
-    query.bool = {should: []};
-
-    controllers.forEach(controller => {
-      query.bool.should.push({exists: {field : 'controllers.' + controller}});
+RoleRepository.prototype.searchRole = function roleRepositorySearchRole (controllers) {
```

Comment:
I have counter-arguments: 
* the default is set to 10 IIRC, and it seems quite a low limit to me
* it is implemented, it works, and it may be needed: what's the need behind this feature removal?